Wallapop Architecture/View.xctemplate/___FILEBASENAME___.swift

```diff
@@ -1,26 +1,30 @@
-import UIKit
 import CommonUI
+import ComposableArchitecture
 
 protocol ___VARIABLE_sceneName___ViewDelegate: AnyObject {
 }
 
-final class ___VARIABLE_sceneName___View: View {
-    private enum AccessibilityIdentifier: String, AccessibilityIdentificable {
-        typealias container = ___VARIABLE_sceneName___View
-        case placeholder
-    }
-    
+final class ___VARIABLE_sceneName___View: View, StatefulView {
     private enum Constant {
     }
     
     weak var delegate: ___VARIABLE_sceneName___ViewDelegate?
    
     override func setupView() {
         backgroundColor = Stylesheet.color(.white)
+        addSubviews()
+    }
+    
+    private func addSubviews() {
+        [].forEach { addSubview($0) }
```

Comment:
We can use shorter version in each places where we iterate through array of subviews in this PR

```suggestion
        [].forEach.(addSubview)
```